android/app/src/main/res/values/strings.xml

```diff
@@ -2196,4 +2196,5 @@
 	<string name="type.amenity.events_venue">Events Venue</string>
 	<string name="type.shop.auction">Auction</string>
 	<string name="type.shop.collector">Collectables</string>
+  <string name="show_on_map">View On Map</string>
```

Comment:
This translation is already present: search_show_on_map

Reply:
Thankyou, I didnt saw that i will use it